### PR TITLE
Create JSON versions of the questions 

### PR DESCRIPTION
**Pull request by bpmutter**
Created JSON versions of questions in 10 languages. Included are the JSON files, the parser functions used to create the JSON, and the .txt source files which are slightly modified version of the .md files viewable on the repo.

### CODE REVIEW

File: JSON/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "id": 38,
+    "question": "What's the output?",
+    "codeSnippet": "(() => {\n  let x, y;\n  try {\n    throw new Error();\n  } catch (x) {\n    (x = 1), (y = 2);\n    console.log(x);\n  }\n  console.log(x);\n  console.log(y);\n})();",
+    "answerOptions": {
+        "A": "`1` `undefined` `2`",
+        "B": "`undefined` `undefined` `undefined`",
+        "C": "`1` `1` `2`",
+        "D": "`1` `undefined` `undefined`"
+    },
+    "correctAnswer": "A",
```
Comment (jakeherp) [inline JSON/README.md:33]: I would prefer answers as an Array
```suggestion
    "answerOptions": [
        "`1` `undefined` `2`",
        "`undefined` `undefined` `undefined`",
        "`1` `1` `2`",
        "`1` `undefined` `undefined`"
    },
    "correctAnswer": 0,
```
or an array of objects:
```suggestion
    "answerOptions": [
      {
        "answer": "`1` `undefined` `2`",
        "correct": true
      },
      {
        "answer": "`undefined` `undefined` `undefined`",
        "correct": false
      }
      {
        "answer": "`1` `1` `2`",
        "correct": false
      },
      {
        "answer": "`1` `undefined` `undefined`"
        "correct": false
      }
    ],
```

**Review by jakeherp — COMMENTED** [review]
I'm not sure if @lydiahallie has a preference on this, but I am not sure if it semantically makes sense to have this as part of this repository, or if this should live on its own and link back to the questions. Alternatively this could live on a separate branch, but since it duplicates questions even further (from md files to txt files to json files, it is practically impossible to keep up-to-date with our `master` branch.

Either way, I have left some feedback below, how I'd personally prefer the data being formatted:

**Comment by bpmutter** [comment]
@jakeherp  i think all your points make complete sense. i've actually already implemented some of the points you brought up since making this pull request. 

namely, i created a separate website with a trivia game API that you can use to either play the trivia game and a public API you can use to query an updated version of the data. You can view the site and access the source code here: 
* [javascript-trivia.com](http://javascript-trivia.com/)
* [frontend API docs + game repo](https://github.com/bpmutter/javascript-trivia)
* [backend API repo](https://github.com/bpmutter/js-trivia-api)

i also identified a couple of bugs in a few questions for the version presented in this pull request, so i would **not** advocate merging this version into the master.  if you / @lydiahallie  would like to add the updated JSON files, i would happily push those up to be added to this repo.

**Comment by jakeherp** [comment]
I am going to close this PR, since we won't be implementing the JSON format as part of this repo.

**Pull request closed without merging**